sdk/schemaregistry/Azure.Data.SchemaRegistry/samples/Sample02_Serialization.md

```diff
@@ -0,0 +1,216 @@
+# Using the `SchemaRegistrySerializer`
+
+The following shows how to use the `SchemaRegistrySerializer` to serialize and deserialize JSON payloads containing Schema Registry schema identifiers and serialized data. The default serialization is JSON, but the serializer is customizable to any type of serialization.
+
+## Table of contents
+- [Implementing `SchemaValidator`](#implementing-schemavalidator)
+- [Serialize and deserialize JSON data using the Event Hubs `EventData` type](#serialize-and-deserialize-json-data-using-the-event-hubs-eventdata-type)
+- [Serialize and deserialize JSON data using `MessageContent` directly](#serialize-and-deserialize-json-data-using-messagecontent-directly)
+- [Pluggable object serialization](#pluggable-object-serialization)
+- [Troubleshooting](#troubleshooting)
+- [Next steps](#next-steps)
+- [Contributing](#contributing)
+
+## Implementing `SchemaValidator`
+
+The `SchemaValidator` is an abstract class that must be implemented and passed into the `SchemaRegistryJsonSerializer` constructor. This allows you to choose the third-party JSON Schema package you would like to use to generate schemas from .NET types and validate .NET objects against JSON schemas. Implementing both `SchemaValidator.Validate` and `SchemaValidator.GenerateSchema` is required.
```

Comment:
You are entirely welcome to disregard this comment, because this sample is very high quality. :)  Since I am 100% new to this domain, I would find a one-sentence description of why someone would want to validate a schema and therefore provide an implementation of the abstract class helpful.  That then gives me a frame of reference to understand the details that follow.  This could also live in the README or somewhere higher-level than this sample.  One could also argue that I need to go read the service docs, which is why you should consider this comment entirely non-blocking :)